### ISSUE TITLE
Support Ruby 2.6.0-preview2 and Rails 5.2.1.

### ISSUE THREAD

**Issue by phyllisstein** [bear-metal/tunemygc#51]
Hey there! After quite a struggle with Kubernetes' container termination patterns, I've finally managed to get TuneMyGC to sync my app's snapshots. Which is very exciting, but for the fact that I get an error message on sync:

```
Ruby version 2.6.0 or Rails version 5.2.1 not supported. Failed to sync 24 snapshots
```

Are there any plans to support Ruby 2.6.0 and/or Rails 5.2.1 in the near future? Thanks in advance for your guidance!

**Comment by methodmissing** [comment]
@phyllisstein it's in the pipeline - I'll try to get through it this week. Which is higher priority for you?

* ruby 2.6
* Rails 5.2.1

?

**Comment by phyllisstein** [comment]
Thanks so much! I guess my personal preference depends on what the current supported versions are, but off the top of my head I'd probably say Rails.

**Comment by donv** [comment]
Supporting Rails 5.2 would be the highest priority for me since Ruby 2.6 is not yet released and Rails 5.2 has been out for a while.

**Comment by ezekg** [comment]
Any update on this? Would love to take this gem for a spin, but I'm running on Rails 5.2 (w/ Ruby 2.5).

**Comment by kreintjes** [comment]
Does this gem/Heroku add-on currently work with Ruby 2.6.2 and Rails 5.2.2?

We used this gem in the past, about two years ago, and then it worked fine. However, we suddenly seem to have lost our GC configuration (stored in ENV variables) and starting new recordings doesn't work (the new recordings don't show up, not even when stopping the measurement). Might also have to do something with the upgrade to heroku-18 stack.

@methodmissing is this gem still supported with new versions of Rails, Ruby, Heroku and Puma (https://github.com/bear-metal/tunemygc/issues/32, https://github.com/bear-metal/tunemygc/issues/49)?

**Comment by jkeen** [comment]
yeah this doesn't seem to be working for me at all on heroku with rails 5.2 and ruby 2.6.5. Fails silently, no error message, just says as soon as data is available I'll be able to manage suggested configurations.
